content/docs/command-reference/plots/index.md

```diff
@@ -98,72 +100,50 @@ names in the `train` array below:
 
 ## Plot templates (data series only)
 
-Users have the ability to change the way data-series plots are displayed by
-modifying the [Vega-Lite specification](https://vega.github.io/vega-lite/), thus
-generating plots in the style that best fits the their needs. This keeps
-<abbr>DVC projects</abbr> programming language agnostic, as it's independent
-from user display configuration and visualization code.
+DVC uses [Vega-Lite](https://vega.github.io/vega-lite/) JSON specifications to
+create visualizations out of users data. Pre-defined specifications are called
+templates.
```

Comment:
```suggestion
DVC uses [Vega-Lite](https://vega.github.io/vega-lite/) JSON templates to
create visualizations out of users' data.
```

Optional suggestion. I don't know that we need to explain what a template means.

Reply:
I'd keep "specifications" since it's the term used in the linked page (explains what templates are but very implicitly). I'd also pull a sentence from the following paragraph here:

```suggestion
DVC uses [Vega-Lite](https://vega.github.io/vega-lite/) JSON specifications
to create plots from user data. Built-in _plot templates_ are stored within the
[`dvc-render`](https://github.com/iterative/dvc-render) library.
```